context/app/static/js/shared-styles/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import Autocomplete, { AutocompleteProps } from '@mui/material/Autocomplete';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { InputLabelProps as inputLabelProps } from '@mui/material/InputLabel';
+
+interface MultiAutocompleteTypes extends AutocompleteProps {
+  tagComponent: React.ReactNode;
+  renderInputProps: TextFieldProps;
+}
+
+interface RenderInputTypes {
+  InputLabelProps: inputLabelProps;
+  textFieldProps: TextFieldProps;
+}
+
+function MultiAutocomplete({ renderInputProps, tagComponent: TagComponent, ...rest }): MultiAutocompleteTypes {
```

Comment:
The `MultiAutocompleteTypes` should be inside the parentheses in order to type the props. If the type is written outside the parentheses, it marks the return type of the function.

```suggestion
function MultiAutocomplete<Value>({ renderInputProps, tagComponent: TagComponent, ...rest }: MultiAutocompleteTypes<Value>) {
```